test/extended/apibinding/apibinding.go

```diff
@@ -105,4 +104,43 @@ var _ = g.Describe("[area/apiexports]", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(output).To(o.ContainSubstring("wildwest.dev/v1alpha1"))
 	})
+
+	g.It("Author:zxiao-Critical-[API] Verify that the bind apiexport command works as expected", func() {
```

Comment:
@zimo-xiao I think title should be changed as "Verify that  trying to bind apiexport command on non-existing destination fails" ?

Reply:
Hi @kasturinarra, thanks for the feedback. I don't think this case only tests binding to non-existing cases. As you can see in line 129:
```go
err = k.Run("kcp").Args("bind", "apiexport", workspaceFullName+":"+apiexportName).Execute()
o.Expect(err).NotTo(o.HaveOccurred())
```
it is trying to bind to an existing apiexport.

Reply:
@zxiao-redhat got it, so if that is the case may be the title should be changed to "Verify that trying to bind apiexport command works as expected for custom resource" ?